x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/configuration_form/source_field_section/source_field_section.tsx

```diff
@@ -104,13 +100,13 @@ export const SourceFieldSection = () => {
         {({ label, helpText, value, setValue }) => (
           <EuiFormRow label={label} helpText={helpText} fullWidth>
             <EuiComboBox
+              noSuggestions
               placeholder={i18n.translate(
                 'xpack.idxMgmt.mappingsEditor.sourceExcludeField.placeholderLabel',
                 {
                   defaultMessage: 'path.to.field.*',
                 }
               )}
-              options={excludeComboBoxOptions}
```

Comment:
I didn't see the use for setting `options` here. If I missed something, please let me know.